test/controllers/admin/comfy/v2/dashboard_controller_test.rb

```diff
@@ -155,4 +246,44 @@ class Comfy::Admin::V2::DashboardControllerTest < ActionDispatch::IntegrationTes
 
     assert_response :success
   end
+
+  # Rack Mini-Profiler test cases - START 
```

Comment:
@alis-khadka could you move these mini profiler tests to its own test file?

Reply:
also you need to add a test to verify that the `pp=env` page is only accessible by `global admin`'s

Reply:
> could you move these mini profiler tests to its own test file?
Sure, I will separate these tests into a separate one.

> also you need to add a test to verify that the pp=env page is only accessible by global admin's
Yup, It's underway. The PR is still in progress.